src/mainwindow.cpp

```diff
@@ -275,6 +275,10 @@ MainWindow::~MainWindow()
 
 void MainWindow::initState()
 {
+    /* Shortcut for Copy Selection Payload to Clipboard */
+    copyPayloadShortcut = new QShortcut(QKeySequence("Ctrl+P"), this);
+    connect(copyPayloadShortcut, &QShortcut::activated, this, &MainWindow::onActionAenuConfigCopyPayloadToClipboardTriggered);
```

Comment:
```suggestion
    connect(copyPayloadShortcut, &QShortcut::activated, this, &MainWindow::onActionMenuConfigCopyPayloadToClipboardTriggered);
```

is this a misprint in ui file somewhere?

Reply:
would it be possible for you to fix this misprint?

Reply:
Ok sure. Will fix it.